packages/components/format-date/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@sl-design-system/format-date",
+  "version": "0.0.0",
+  "description": "Format-date component for the SL Design System",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sl-design-system/components.git",
+    "directory": "packages/components/format-date"
+  },
+  "homepage": "https://sanomalearning.design/components/format-date",
+  "bugs": {
+    "url": "https://github.com/sl-design-system/components/issues"
+  },
+  "type": "module",
+  "main": "./index.js",
+  "module": "./index.js",
+  "types": "./index.d.ts",
+  "customElements": "custom-elements.json",
+  "exports": {
+    ".": "./index.js",
+    "./format-date-time.js": "./src/format-date-time.js",
```

Comment:
`import { formatDateTime } from '@sl-design-system/format-date/format-date-time.js';` is kind of repeating :) In format number, i have `import { format } from '@sl-design-system/format-number/format.js';` Perhaps do the same here?